kubernetes-maven-plugin/doc/src/main/asciidoc/inc/goals/_goals.adoc

```diff
@@ -14,16 +14,16 @@ This plugin supports a rich set for providing a smooth Java developer experience
 |===
 | Goal | Description
 
-| *<<k8s:build>>*
+| *<<jkube:build>>*
```

Comment:
I think `{plugin-prefix}` parameter should be used here

Reply:
That's just a reference, it will get converted when rendered to whatever that reference is pointing to:
![image](https://user-images.githubusercontent.com/488391/81061873-64e30800-8ed5-11ea-8133-0218e108dc18.png)